service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/AzureStorageAccessService.java

```diff
@@ -247,4 +263,59 @@ public BlobContainerClient buildBlobContainerClient(
         .containerName(containerResource.getStorageContainerName())
         .buildClient();
   }
+
+  private StorageData getStorageAccountData(
```

Comment:
There is similar code in https://github.com/DataBiosphere/terra-workspace-manager/blob/688c3f42db57fcb1206f2a3c3a45ff6d1d12f85e/service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/storageContainer/VerifyAzureStorageContainerCanBeCreatedStep.java (see `validateStorageAccountExists`). Can we have a shared method that gets the parent storage account of a container?

Reply:
Agree. Code is similar in some way. It's probably good place for improvement. I think that this improvement might change the scope of current ticket. I created separate Jira ticket for this - TOAZ-278.